diff -ura base-3.15.5.old/startup/Site.cshrc base-3.15.5/startup/Site.cshrc
--- base-3.15.5.old/startup/Site.cshrc	2018-12-13 14:09:11.331734355 -0800
+++ base-3.15.5/startup/Site.cshrc	2018-12-13 14:12:53.191727460 -0800
@@ -14,12 +14,12 @@
 
 # Location of epics base
 if ( ! $?EPICS_BASE ) then
-	set EPICS_BASE=/usr/local/epics/base
+	set EPICS_BASE=/opt/epics/base
 endif
 
 # Location of epics extensions
 if ( ! $?EPICS_EXTENSIONS ) then
-	setenv EPICS_EXTENSIONS /usr/local/epics/extensions
+	setenv EPICS_EXTENSIONS /opt/epics/extensions
 endif
 
 # Postscript printer definition needed by some extensions (eg medm, dp, dm, ...)
diff -ura base-3.15.5.old/startup/Site.profile base-3.15.5/startup/Site.profile
--- base-3.15.5.old/startup/Site.profile	2018-12-13 14:09:11.331734355 -0800
+++ base-3.15.5/startup/Site.profile	2018-12-13 14:11:48.555062806 -0800
@@ -14,12 +14,12 @@
 
 # Location of epics base
 if [ -z "${MY_EPICS_BASE}" ] ; then
-	MY_EPICS_BASE=/usr/local/epics/base
+	MY_EPICS_BASE=/opt/epics/base
 fi
 
 # Location of epics extensions (medm, msi, etc.)
 if [ -z "${EPICS_EXTENSIONS}" ] ; then
-	EPICS_EXTENSIONS=/usr/local/epics/extensions
+	EPICS_EXTENSIONS=/opt/epics/extensions
 fi
 
 # Postscript printer definition needed by some extensions (eg medm, dp, dm, ...)
